### ISSUE TITLE
Add API endpoint to query unit count of the current session

### ISSUE THREAD

**Issue by elsirion** [fedimint/fedimint#4185]
This can be useful for debugging consensus problems. Thoughts @joschisan?

**Comment by joschisan** [comment]
You can query the SESSION_STATUS_ENDPOINT for this.

**Comment by elsirion** [comment]
No, it isn't. Unit count is useful for debugging in case no consensus intems are being produced since units are being produced regardless.

**Comment by joschisan** [comment]
A unit is technically a node in the directed acyclic graph that's is produced by aleph bft for every session and continuously grows while the session runs. These units have a batch of consensus items attached each; as the graph grows we use its structure to derive consensus on an ordering on its units and therefore the attached batches which implicitly orders the consensus items. Now, after consensus item has been ordered its becomes a accepted item if and only if fedimint consensus accepts it as valid.
